frontend/event/eventController.js

```diff
@@ -211,8 +211,42 @@
                 eventCtrl.selectedYear = new Date().getFullYear();
                 eventCtrl._loadYears();
                 eventCtrl.loadMoreEvents();
+                eventCtrl.toolbarItems = getToolbarMobileMenuItems();
             });
         };
+        
+        const toolbarMobileMenuItems = [];
+        
+        function getToolbarMobileMenuItems() {
+            const monthsMenuItem = {
+                options: eventCtrl.months.map(month => month.month_name),
+                action: month => { eventCtrl.selectedMonth = eventCtrl.months.find(m => m.month_name == month); eventCtrl.loadFilteredEvents(); },
```

Comment:
I didn't understand what u mean with the first part. I'm not using the object? If you're talking about the month's object, that's because to be generic I get only its name, once the year is not an object and the notification options are not objects either. Yes, i'm searching in the array every time, but this operation is O(1) once the array has a fixed size = 12